src/components/ContentForm.jsx

```diff
@@ -0,0 +1,204 @@
+import React, { useState, useRef } from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+
+import Wrapper from "./Wrapper";
+import ImgWrapper from "./ImgWrapper";
+import HeartCounter from "./HeartCounter";
+import Button from "./SButton";
+
+const Input = styled.input`
+  width: 205px;
+  border-radius: 5px;
+  border: solid 1px transparent;
+
+  :focus {
+    outline: none;
+  }
+`;
+
+const HeartInputWrapper = styled.div`
+  display: inline-flex;
+  width: 90px;
+  flex-direction: column;
+  align-items: center;
+  input {
```

Comment:
```suggestion

  input {
```

여기부터는 선택자니까 한줄 띄어줘도 좋을 것 같습니다! 스타일린트 설정도 좀 더 디테일하게 하는 게 좋을 것 같기도 하네요

Reply:
css in js에서 stylelint는 많은 일을 못하기 때문에 너무 기대하지는 마세용...